test/util.inc

```diff
@@ -17,23 +17,41 @@
 #
 #   $1 the directory containing the CMake simulation code
 function run_cmake_simulation() {
+  if [ -n "$ZSH_VERSION" ]; then
```

Comment:
Please use $SHELL and turn this into a case statement, so that in the future, this can be expanded to other shells. See https://github.com/Homebrew/install/blob/master/install.sh#L636

Reply:
It seems ``$SHELL`` denotes the login shell, and is set by the login process. It doesn't indicate the shell _currently_ executing the script. It may also be unset. This is fine in the brew install script as it's used for populating the profile. In our case, we need to know the shell currently executing `thisbdm.sh`.